examples/erosion2/storage.py

```diff
@@ -1,33 +1,64 @@
 import shelve
+import string
+import time
+
+
+_CHARS = sorted(string.ascii_lowercase + string.digits)
+_N_CHARS = len(_CHARS)
+
+
+def _encode_id(num):
```

Comment:
How about adding a couple doctests?

Reply:
That function I've copied from the original erosion app. I have to work on it to understand what it does :) But sure.

Reply:
Haha in that case, allow me:

```python
>>> link_map.id_encode(41660)
'z0a'
>>> link_map.id_encode(4166000)
'2s5tk'
>>> link_map.id_encode(41660 + int(1e7))  # 10 million links later
'6p79k'
```

Reply:
Great, thanks. I think I get what it does, it's a simple base-35 (?) encoding with an extended alphabet.